test/unit/pdf/finance/invoice_report_data_test.rb

```diff
@@ -113,12 +113,15 @@ def setup
     default_options = Paperclip::Attachment.default_options
     Paperclip::Attachment.stubs(default_options: default_options.merge(storage: :s3))
     @provider.profile.update(logo: Rack::Test::UploadedFile.new(file_fixture('wide.jpg'), 'image/jpeg', true))
-    URI.expects(:open).with(regexp_matches(/\Ahttps.*\/profiles\/logos\/invoice\/wide.png\z/)).returns(File.open(file_fixture('wide.jpg')))
+
+    # the extension in the request is .png, because images are converted to png in the 'invoice' style
+    stub_request(:get, %r{\Ahttps.*/profiles/logos/invoice/wide.png\z})
+        .to_return(status: 200, body: File.open(file_fixture('wide.jpg')))
```

Comment:
This will no loner be the case after #3065 

But I guess I will have to fix it with it :(

Reply:
Yes... we'll need to review this afterwards...

Reply:
there is no migration issue, just fix the test after